examples/meanio/README.md

```diff
@@ -0,0 +1,265 @@
+```
+
+If you ssh to that machine, you can run `docker ps` to see the actual pod:
+
+```shell
+me@workstation$ gcloud compute ssh kubernetes-minion-yxea
+
+me@kubernetes-minion-yxea:~$ sudo docker ps
+CONTAINER ID        IMAGE                                  COMMAND                CREATED              STATUS              PORTS                    NAMES
+efabcc7f7da9        mongo:2                                "/entrypoint.sh mong   23 hours ago        Up 23 hours                                                          k8s_meanio-mongodb.e5bffb35_meanio-mongodb-controller-nyxxv.default.api_499dd2a6-ce04-11e4-94ac-42010af070a6_d9391efe
+```
+
+(Note that initial `docker pull` may take a few minutes, depending on network conditions.  You can monitor the status of this by running `journalctl -f -u docker` to check when the image is being downloaded.  Of course, you can also run `journalctl -f -u kubelet` to see what state the kubelet is in as well during this time.
+
+### Step Two: Fire up the MongoDB service
+A Kubernetes 'service' is a named load balancer that proxies traffic to *one or more* containers. This is done using the *labels* metadata which we defined in the meanio-mongodb pod above.
+
+The services in a Kubernetes cluster are discoverable inside other containers via environment variables.
+
+Services find the containers to load balance based on pod labels.
+
+The pod that you created in Step One has the label `name=meanio-mongodb`. The selector field of the service determines *which pods will receive the traffic* sent to the service, and the port and containerPort information defines what port the service proxy will run at. 
+
+Use the file `examples/meanio/meanio-mongodb-service.json`:
+
+```js
+{
+  "id": "meanio-mongodb",
+  "kind": "Service",
+  "apiVersion": "v1beta1",
+  "port": 27017,
+  "containerPort": 27017,
+  "labels": {
+    "name": "meanio-mongodb"
+  },
+  "selector": {
+    "name": "meanio-mongodb"
+  }
+}
+```
+
+to create the service by running:
+
+```shell
+$ kubectl create -f examples/meanio/meanio-mongodb-service.json
+meanio-mongodb
+
+$ kubectl get services
+NAME                    LABELS                                    SELECTOR                     IP                  PORT
+meanio-mongodb          name=meanio-mongodb                                             name=meanio-mongodb          10.0.222.168        27017
+```
+
+This will cause all pods to see the MongoDB server apparently running on <ip>:27017.
+
+- Traffic will be forwarded from the service "port" (on the service node) to the  *containerPort* on the pod which (a node the service listens to). 
+
+Thus, once created, the service proxy on each minion is configured to set up a proxy on the specified port (in this case port 27017).
+
+
+make sure that the mongodb pod is running and listening by running:
+
+```shell
+
+$ kubectl log meanio-mongodb-controller-nyxxv
```

Comment:
Can you please add `meanio-mongodb` to the end of this command to alleviate #5861? It will become

```shell
$ kubectl log meanio-mongodb-controller-nyxxv meanio-mongodb
```